doc/api/test.md

```diff
@@ -1368,6 +1368,11 @@ machines or processes, ideal for large-scale executions across varied
 environments. It's incompatible with `watch` mode, tailored for rapid
 code iteration by automatically rerunning tests on file changes.
 
+**Note:** A number of CLI flags will not function properly when used with
+`execArgv`, and users should avoid using it unless they have a deep
+understanding of its implications. In general, bug reports regarding this
+behavior will not be investigated.
```

Comment:
I think we have the list somewhere, right? We should link it.

Reply:
Last time I checked I don't think we did.

Reply:
If not, we should probably add one or this note really is not going to be very useful for folks.